### DISCUSSION TITLE
[Support]: Issue RAM Leaking since some weeks

### DISCUSSION THREAD

**Discussion by LiloBzH** [blakeblackshear/frigate#11383] [General Support]
### Describe the problem you are having

Hi

For several weeks, Frigate has been constantly consuming RAM to finally saturate the server's total RAM.

In another ticket, it was said that it is the fault of PYTHON and an update but why are HOME ASSISTANT, ChangeDetection which also use Python not impacted?

The memory limit in the container does not work on my side.
Memory consumption increases without stopping

Thank you !

### Version

13.2

### Frigate config file

```yaml
no need
```

### Relevant log output

```shell
no need
```

### FFprobe output from your camera

```shell
no need
```

### Operating system

Proxmox

### Install method

Docker Compose

### Coral version

M.2

### Network connection

Wired

### Camera make and model

3rd

====================

Month and Week 

![2024-05-15_14-59-03_896b2e77-ae1c-49a5-827f-215b5aa963a6](https://github.com/blakeblackshear/frigate/assets/4649466/f782e07d-d021-40e3-8952-a933ef059949)

![2024-05-15_14-59-12_ffe58a64-9928-4ae9-968c-e7098427b531](https://github.com/blakeblackshear/frigate/assets/4649466/88c4f61d-aacc-4d94-8b91-63f779c4e2c6)

Today 

After reboot of proxmox, i dont start FRIGATE avec memory don't move

![2024-05-15_14-59-22_572bf9c1-dcb8-4f62-951e-e2a06975e626](https://github.com/blakeblackshear/frigate/assets/4649466/ebdf8079-23b9-4e2e-b056-c31040c52573)

**Comment by LiloBzH** [comment]
Hey, I just discovered that a test is in progress by deactivating the option:

hwaccel_args: preset-vaapi

And more generally video acceleration basically

test in process

**Comment by DSew20** [comment]
Did you ever find a fix for this issue? I have a similar memory leak issue on Proxmox causing it to crash a few hours every restart.
